feat: Add LW5 Community Day Blogpost. (#8522)
supabase/supabase: apps/www/_blog/2022-08-19-launch-week-5-community-day.mdx

<blockquote class="twitter-tweet" data-theme="dark">
  <p lang="en" dir="ltr">
    Thanks to <a href="https://twitter.com/supabase?ref_src=twsrc%5Etfw">@supabase</a> for this
    little beauty (the hat, not the cat).
    <br />
    <br />
    Now I’ve just got to pry it away from Peanut 🥜 😂 <a href="https://t.co/POBdVZrF5N">
      pic.twitter.com/POBdVZrF5N
    </a>
  </p>
  &mdash; Daniel Cranney 🇬🇧 (@danielcranney) <a href="https://twitter.com/danielcranney/status/1557249589182447616?ref_src=twsrc%5Etfw">August 10, 2022</a>
</blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

_Our limited edition swag is approved by professional swag testers like Peanut!_

### Launch Week Landing Page & Golden Tickets

<iframe
  className="w-full"
  width="560"
  height="315"
  src="https://www.youtube-nocookie.com/embed/4_duDgQeCyM?modestbranding=1&autohide=1&showinfo=0&controls=0"
  title="YouTube video player"
  frameborder="0"
  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
  allowfullscreen
></iframe>

> We now have a shiny new l[aunch week page](http://supabase.com/launch-week), your one-stop shop for all things Launch Week, where you can find each day’s feature as well as highlights from the past Launch Weeks.

Reviewer: /nit Should it be [launch week page]?

Author: fixed thanks

> We now have a shiny new l[aunch week page](http://supabase.com/launch-week), your one-stop shop for all things Launch Week, where you can find each day’s feature as well as highlights from the past Launch Weeks.

Author: ```suggestion
We now have a shiny new [launch week page](http://supabase.com/launch-week), your one-stop shop for all things Launch Week, where you can find each day’s feature as well as highlights from the past Launch Weeks.
```

We've added a Golden Ticket mechanic, distributing some special edition launch week swag to some lucky winners. If you're interested in how we've built this, check out the video above with a brief explanation.

---

## Open Source Spotlight

### pg_jsonschema

<iframe
  className="w-full"
  width="560"
  height="315"
  src="https://www.youtube-nocookie.com/embed/4A6RKPp1b70?modestbranding=1&autohide=1&showinfo=0&controls=0"
  title="YouTube video player"
  frameborder="0"
  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
  allowfullscreen
></iframe>

For our Open Source Spotlight we're hearing from [Dmitry Dygalo](https://twitter.com/Stranger6667) from [schemathesis.io](https://schemathesis.io/) who works on a multitude of open-source projects. He's the author and maintainer of [jsonschema-rs](https://github.com/Stranger6667/jsonschema-rs), which is the foundation for our new [pg_jsonschema extension](https://github.com/supabase/pg_jsonschema), a PostgreSQL extension providing JSON Schema validation.

> ### Postgrest v10: Native Types
> 
> Today, PostgREST v10 was [released](https://github.com/PostgREST/postgrest/releases/tag/v10.0.0).
> In this [blog post](/blog/postgrest-v10) we explore all the new features: EXPLAIN, Query Planners, Improved Relationship Detection, and Computed relationships.
> 
> [Read the blog post](/blog/postgrest-v10).

Reviewer 2: ```suggestion
### Postgrest v10

Today, PostgREST v10 was [released](https://github.com/PostgREST/postgrest/releases/tag/v10.0.0).
In this [blog post](/blog/postgrest-v10) we explore some of its new features: EXPLAIN and Improved Relationship Detection.

[Read the blog post](/blog/postgrest-v10).
```
